### ISSUE TITLE
Sound under Linux don't sound very well

### ISSUE THREAD

**Issue by GoogleCodeExporter** [leniad/dsp-emulator#29]
```
The sound under linux sounds very bad.
Clicks all the time and is delayed about 1 sec
```

Original issue reported on code.google.com by `leni...@hotmail.com` on 7 Mar 2012 at 12:51

**Comment by leniad** [comment]
Waiting for a new SDL2 release.

**Comment by leniad** [comment]
Fixed on DSP 0.16b2
